docs/data/material/getting-started/support/support.md

```diff
@@ -66,12 +66,13 @@ Find details on the [supported versions](/versions/#supported-versions).
 MUI X comes in [three plans](https://mui.com/pricing/).
 The support available on the Community plan is mostly made possible thanks to people like you: the community. It's described in more detail in the above section. The maintainers of MUI focuses on solving the root issues, not direct support.
 
-The Pro plan allows developers to license extra features that are challenging to find in the OSS space.
-The maintainers of MUI provide support for these paid features.
-They aim for a similar quality of experience to the one provided by the community on the open-source modules
-however no SLAs are provided.
+The Pro plan allows developers to license components and extra features that are challenging to find in the OSS space.
+To provide a similar quality of experience to the one provided by the community on the MUI Core modules,
+the maintainers of MUI provide support for the advanced components included in MUI X.
```

Comment:
We could, but I don't think that the reason mentioned is correct. It's not because MUI X operates independently from the Core team that we don't do it. I think that we don't do it because it's not what we want to sell in this plan, I suspect it would require a different pricing structure.

Reply:
```suggestion
To provide a similar quality of experience to the one provided by the community on the MUI Core modules,
the maintainers of MUI provide support for the advanced components included in MUI X.

This plan includes direct technical support from MUI X maintainers to MUI X components exclusively. 
Technical support for MUI Core components is not included.
```

Reply:
Suggestion applied.